implementations/rust/c/bindings/src/ctypes.rs

```diff
@@ -0,0 +1,10 @@
+#![allow(non_camel_case_types)]
```

Comment:
Not sure if this is necessary with [this](https://doc.rust-lang.org/std/os/raw/index.html).
Can we use the built-in rust ones instead?

Reply:
Yes, fixed